src/Command/Docker/Build.php

```diff
@@ -170,100 +95,31 @@ protected function configure()
                 self::OPTION_MODE,
                 'm',
                 InputOption::VALUE_REQUIRED,
-                sprintf(
-                    'Mode of environment (%s)',
-                    implode(
-                        ', ',
-                        [
-                            ComposeFactory::COMPOSE_DEVELOPER,
-                            ComposeFactory::COMPOSE_PRODUCTION,
-                            ComposeFactory::COMPOSE_FUNCTIONAL,
-                        ]
-                    )
-                ),
-                ComposeFactory::COMPOSE_PRODUCTION
+                sprintf('Mode of environment')
             )->addOption(
                 self::OPTION_SYNC_ENGINE,
                 null,
                 InputOption::VALUE_REQUIRED,
-                sprintf(
-                    'File sync engine. Works only with developer mode. Available: (%s)',
-                    implode(', ', DeveloperCompose::SYNC_ENGINES_LIST)
-                ),
-                DeveloperCompose::SYNC_ENGINE_DOCKER_SYNC
+                sprintf('File sync engine. Works only with developer mode.')
```

Comment:
Looks like using `sprintf` is redundant in this place

Reply:
Fixed